src/main/java/seedu/address/model/property/PropertyTagsPredicate.java

```diff
@@ -0,0 +1,45 @@
+package seedu.address.model.property;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.function.Predicate;
+
+import seedu.address.model.tag.Tag;
+
+/**
+ * Tests that a {@code Property}'s {@code Tags} contains the tags given.
+ */
```

Comment:
Perhaps this can be clearer by saying `... contains all of the tags given`? 

Side note: should this be changed to contains any of the tags given instead? Since the user might want to use a few tags to search for properties with any of them.

Reply:
Alright sure. Anyway I just realised a user can search for containing all given tags by using multiple r/ parameters.